pkg/reconciler/broker/broker.go

```diff
@@ -53,6 +55,11 @@ import (
 	"knative.dev/eventing/pkg/reconciler/names"
 )
 
+const (
+	brokerIngressTLSSecretName = "mt-broker-ingress-server-tls"
+	caCertsSecretKey           = "ca.crt"
```

Comment:
the `ca.crt` key is created multiple times in different places (now and in the previous PR), can we have a constant like `TLSCaCrt` used across multiple components in the `eventingtls.go` file? similar to
https://github.com/knative/eventing/blob/f92a05b72d21b7effd8e5bac9d1572962bd47e4f/pkg/eventingtls/eventingtls.go#L40-L43